MANIFEST.in

```diff
@@ -1 +1,2 @@
 graft mira/dkg/templates
+include mira/dkg/metaregistry/epi.json
```

Comment:
this should be covered by `include_package_data = True` in `setup.cfg` - is this not the case?

Reply:
no, it is missing unless I add it here